projects/pastanaga-angular/src/lib/table/table-cell-header-sortable/table-cell-header-sortable.component.ts

```diff
@@ -8,25 +8,23 @@ import { Sort } from '../../common';
     styleUrls: [],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TableCellHeaderSortableComponent implements OnInit {
+export class TableCellHeaderSortableComponent {
     @Input()
     set reversed(value: boolean) {
         this._reversed = coerceBooleanProperty(value);
+        if (this._reversed) {
+            this.icon = 'sorted-ascending';
+            this.currentSort = Sort.ascending;
+        }
```

Comment:
here you manage only one case, but you need to manage all cases (imagine reversed property is changed multiple times).
```typescript
this.icon = this._reversed ? 'sorted-ascending' : 'sorted-descending';
this.currentSort = this._reversed ? Sort.ascending : Sort.descending;
```